### PR TITLE
Package dateadd update and models description translation

### PR DESCRIPTION
**Pull request by eduardo-nercolini**
### Overview

This PR proposes replacing the dbt_utils.dateadd function from dbt_utils with dbt.dateadd from dbt core. The reason is that the former function has been deprecated in newer versions of dbt_utils and doesn't work in most recent dbt_utils packages (>=0.9.0). Additionally, the surrogate_key function has been renamed to generate_surrogate_key for the same reason. Furthermore, it includes translating the descriptions of models and source from Portuguese to English to align with the rest of the package, which is already in English and adding a packages.yml file to specify the packages version accepted (also written in the Readme text).

### Key changes

- Dimensions and Fact tables changed from dbt_utils.surrogate_key to dbt_utils.generate_surrogate_key;
- Staging tables changed from dbt_utils.dateadd to dbt.dateadd;
- source, dimensions and fact profiles tables description translated to english;
- added packages.yml;
- Reamde.md updated.

### Other notes

Once this PR is merged, it will work as a new release, therefore, tag v1.0.0.

### Checklist
_The following items are not covered by automated testing._
- [N/A] All added/modified models and columns are documented and tested in `schema.yml` files.
- [X] All tests pass. **OR**    
    _Check one:_
    - [ ] There are no new test failures, whether or not the existing model was intentionally changed for this PR.
    - [ ] This PR causes new test failures and there is a plan in place for addressing them.

### CODE REVIEW

**Comment by eduardo-nercolini** [comment]
@jessicaoblira alguma posição sobre o PR e demais modificações? Estou no aguardo para poder passar isso pro projeto do cliente. Obrigado.

File: models/staging/dbt_utils_day.sql [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {% set my_query %}
-    select cast({{dbt_utils.current_timestamp()}} as date)
```
Comment (vitoravancini) [inline models/staging/dbt_utils_day.sql:2]: @eduardo-nercolini  wont this break compatibilty in diferent sql engines?

File: models/staging/dbt_utils_day.sql [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    select cast({{current_timestamp()}} as date)
 {% endset %}
 
 {% if execute %}
     {% set today = run_query(my_query).columns[0].values()[0] %}
-    {% set tomorrow = dbt_utils.dateadd('day', 1, "'" ~ today ~ "'") %}
+    {% set tomorrow = dbt.dateadd("day", 1, "'" ~ today ~ "'") %}
```
Comment (vitoravancini) [inline models/staging/dbt_utils_day.sql:7]: do this work? ''dbt''?

**Review by vitoravancini — CHANGES_REQUESTED** [review]
are we upgrading dbt utils?

File: models/staging/dbt_utils_day.sql (hunk 1 above)
Comment (eduardo-nercolini) [reply models/staging/dbt_utils_day.sql:2]: It's a good question. I would say that the `current_timestamp()` function is a standard SQL function and is widely supported across most SQL database engines, and it's unlikely to "break" in these engines in the sense of causing errors or unexpected behavior due to its widespread adoption and conformity to SQL standards. However, there might be some cases that I'm not aware of where its behavior could cause some errors. I'm open to suggestions for addressing this issue.

**Pull request merged**